docs/hoon/concepts.md

```diff
@@ -105,52 +137,21 @@ which makes your program more readable anyway.  Also, the dumber
 the compiler, the easier it is for a dumb human to understand
 what the compiler is thinking.
 
-### <a name="twig">`twig`</a> (expression)
-
-A [`twig`](../reference) is a Hoon AST node, the result of parsing a
-source expression.  A Hoon source file encodes one twig.
-
-As the noun that the parser produces, a twig is a tagged union of
-the form `{stem bulb}`, where `stem` (the tag) is a `cord` which
-controls the type of `bulb` (the data).
-
-As text, each twig has both regular and irregular syntax forms.
-Regular syntax can be either *tall* or *flat*: multiline or
-single-line.  Irregular syntax is always flat.   Tall twigs can
-contain flat twigs, but not vice versa.
-
-Regular forms always start with a *sigil*, which is either a
-*keyword* or a *rune* at the programmer's choice.  A keyword is
-the stem cord.
+To compile the hoon from the last subsection into Nock, try the 
+following in `:dojo`:
 
-For most stems, tall regular form has no delimiter or terminator,
-eliminating the `)))))` problem common in functional languages.
-Hoon also uses an unusual "backstep" indentation pattern to
-control left-margin drift.
-
-Hoon is mildly whitespace-sensitive.  It knows the difference
-between no whitespace, one-space, and more-space.  But all cases
-of more-space mean the same thing.  Flat syntax uses one-space,
-tall syntax more-space.
-
-As an example, the regular flat keyword syntax `:call(a b)` means
-the same thing as the flat rune syntax `%-(a b)`, the tall
-keyword syntax
-
-```
-:call  a
-b
 ```
-or the tall rune syntax (here on one line, with double spaces):
-```
-%-  a  b
+(mint:ut %noun [%clhp p=[%sand p=%ud q=2] q=[%sand p=%ud q=17]])
 ```
 
-### <a name="span">`span`</a> (type, as range)
+The `%noun` is type information, and the tagged union after that 
+is the hoon.
+
+### <a name="type">`type`</a> (type, as range)
 
-A `span` is a Hoon type.  It defines a set (finite or
-infinite) of nouns and ascribes some semantics to it.  There is
-no syntax for spans; they are always defined by inference (ie, by
+A `type` defines a set (finite or infinite) of nouns and ascribes 
+some semantics to it.  There is no direct syntax for defining 
+types; they are always defined by inference (ie, by
```

Comment:
ie -> i.e.

Reply:
I would also mention something about how the types are assembled out of base types defined in ++type. The type inference assembles complex types out of the simpler built-in types.